src/main/java/busTickets/FindMax.java

```diff
@@ -0,0 +1,14 @@
+package busTickets;
+
+public class FindMax {
```

Comment:
Class name should be noun, and usually it should differ from method name :)
General purpose: MaxFinder/MaxCalculator/MaxUtil/MaximumUtility